schematic/store/synapse.py

```diff
@@ -50,8 +52,100 @@
 from schematic.exceptions import MissingConfigValueError, AccessCredentialsError
 
 from schematic import CONFIG
+from schematic.utils.general import entity_type_mapping
```

Comment:
Please move this to be next to other `schematic.utils` imports.

Reply:
Yeah sure. I could do that.